web_ui/src/features/dataset-import/hooks/use-local-storage-dataset-import.hook.ts

```diff
@@ -29,7 +29,18 @@ export const useLocalStorageDatasetImport = <T extends DatasetImport>(
     };
 
     const putLsDatasetImport = (item: T): void => {
-        setLsDatasetImports([...(getParsedLocalStorage(storageKey) as T[]).filter(hasDifferentId(item.id)), item]);
+        const currentItems = (getParsedLocalStorage(storageKey) as T[]) || [];
```

Comment:
Instead of changing the hook itself, could we not just let the consumer decide how to present them after retrieving them? So, we can put in whichever order we want, but when getting them, we can filter or sort the way we want.